docs/FAQ.md

```diff
@@ -95,7 +95,7 @@ which may require multiple invocations of `jj rebase -r` or `jj rebase -s`.
 
 To squash or split commits, use `jj squash` and `jj split`.
 
-### How can I keep my scratch files in the repository?
+### How can I create scratch files without comitting them to the repository?
```

Comment:
Nit: I think the previous title was clearer, since it implied that it works _also_ for adding files.

Reply:
What do you mean by adding files?

At a glance, either title seems OK to me, but I also like the idea of combining them:

> How can I keep my scratch files in the working copy without committing them to the repository?

Reply:
I think phrasing questions as someone unfamiliar with jj would ask them might make the FAQ easiest to navigate. I only pondered it for a couple seconds while writing this commit, but my concern with "working copy" was:

- I don't _think_ most developers use that term. They may recognize it in context, but wouldn't use it themselves.
- It might be unclear that we're referring to "the directory tree of the workspace" rather than "the working copy commit".

I'm not attached to any particular phrasing though.

Reply:
My opinion on that is similar, see https://github.com/martinvonz/jj/pull/4320#discussion_r1729209626